.github/workflows/Connector Master SDK Workflow.yml

```diff
@@ -216,18 +230,25 @@ jobs:
       - name: Quality Gate
         id: quality-step
         run: |
-          if "${{ steps.validator-quality-gate.outcome }}" == "failure" or "${{ steps.unit-tests.outcome }}" == "failure" or "${{ steps.sonarcloud-quality-gate-check.outcome }}" == "failure" or "${{ steps.sonarcloud-quality-gate-check.outputs.quality-gate-status }}" == "FAILED":
-            print("Quality gate failed due to:")
-          if "${{ steps.unit-tests.outcome }}" == "failure":
-            print("- Test failures")
-          if "${{ steps.validator-quality-gate.outcome }}" == "failure":
-            print("- Validator issues")
-          if "${{ steps.sonarcloud-quality-gate-check.outcome }}" == "failure":
-            print("- Could not retrieve SonarCloud quality gate status")
-          if "${{ steps.sonarcloud-quality-gate-check.outputs.quality-gate-status }}" == "FAILED":
-            print("- Code analysis quality gate failed")
-          if "${{ steps.validator-quality-gate.outcome }}" == "failure" or "${{ steps.unit-tests.outcome }}" == "failure" or "${{ steps.sonarcloud-quality-gate-check.outcome }}" == "failure" or "${{ steps.sonarcloud-quality-gate-check.outputs.quality-gate-status }}" == "FAILED":
-            exit(1)
+          if "${{ steps.detectCsprojFiles.outputs.csproj-file-present }}" == "false":
+            print("SonarCloud quality gate skipped as no .csproj files were detected.")
+            if "${{ steps.validator-quality-gate.outcome }}" == "failure":
```

Comment:
Not sure if it impacts, but indentation is off for all the other ifs

Reply:
indentation increased because this code is now inside a newly introduced if/else statement